<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgVideoBase</class>
 <widget class="QWidget" name="DlgVideoBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </item>
   <item>
    <widget class="QGroupBox" name="m_pConfGB">
     <property name="title">
      <string>Device Configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="VideoSettings" name="m_pVideoSettings" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="VideoWidget3" name="m_pPreviewGV" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_pPreviewPB">
     <property name="text">
      <string>Start preview</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget3</class>
   <extends>QWidget</extends>
   <header>widgets/videowidget3.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VideoSettings</class>
   <extends>QWidget</extends>
   <header>widgets/videosettings.h</header>
   <container>1</container>
   <slots>
    <signal>settingsChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_pVideoSettings</sender>
   <signal>settingsChanged()</signal>
   <receiver>DlgVideoBase</receiver>
   <slot>slotSettingsChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotSettingsChanged()</slot>
 </slots>
</ui>
